patching/plugin-commands-patching/src/patchCommit.ts

```diff
@@ -127,6 +131,28 @@ export async function handler (opts: PatchCommitCommandOptions, params: string[]
   }) as Promise<undefined>
 }
 
+interface GetPatchContentContext {
+  patchedPkg: ParseWantedDependencyResult
+  patchedPkgDir: string
+  tmpName: string
+}
+
+type GetPatchContentOptions = Pick<PatchCommitCommandOptions, 'modulesDir' | 'storeDir'> & WritePackageOptions
+
+async function getPatchContent (ctx: GetPatchContentContext, opts: GetPatchContentOptions): Promise<string> {
+  const srcDir = opts.storeDir
+    ? path.resolve(opts.storeDir, 'tmp', 'patch-commit', ctx.tmpName)
```

Comment:
storeDir is calculated with the store-path package and the tmp dir is in the versioned subdir.

Reply:
I don't understand what you meant? `opts.storeDir` should already be defined, no? What exactly are you suggesting?

Reply:
https://github.com/pnpm/pnpm/blob/58d859793f8238a6e78aaae57e0727508ad7a29a/store/store-connection-manager/src/index.ts#L31-L35

Reply:
https://github.com/pnpm/pnpm/pull/9251/commits/01a5009f5ac6a74c58eafb418d3b9af802cc8546